packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -502,10 +507,46 @@ You've created a plugin project that doesn't yet support any platforms.
 ''');
 }
 
-void _printAddPlatformMessage(String pluginPath) {
+void _printPluginAddPlatformMessage(String pluginPath) {
   globals.printStatus('''
 To add platforms, run `flutter create -t plugin --platforms <platforms> .` under $pluginPath.
 For more information, see https://flutter.dev/go/plugin-platforms.
 
 ''');
 }
+
+// shows warning if user requested a platform and if it's not enabled
+void _printWarningToEnablePlatforms(
```

Comment:
refactor this method into 2 methods.
1st to return a list of all the disabled platforms that needs to be warned.
2nd to actually print the message.

Reply:
Did this.